modules/nw_multinetwork-sriov.adoc

```diff
@@ -0,0 +1,315 @@
+// admin_guide/multinetwork.adoc
+ifdef::context[:parent-context: {context}]
+
+[id='multinetwork-sriov-{context}']
+= SR-IOV Overview
+
+OpenShift includes the capability to use SR-IOV hardware on OpenShift nodes, which enables you to attach SR-IOV virtual function (VF) interfaces to pods as in addition to other network interfaces.
+
+There are two components required to provide this capability, the SR-IOV network device plugin, and the SR-IOV CNI plugin.
+
+SR-IOV network device plugin is a Kubernetes device plugin for discovering, advertising and allocating SR-IOV network virtual function (VF) resources. Device Plugins are used in Kubernetes to enable the use of the limited resources, typically in physical devices. Device plugins give the Kubernetes scheduler awareness of which resources have been exhausted, allowing pods to be scheduled to worker nodes with sufficient resources available. The SR-IOV CNI plugin plumbs VF interfaces allocated from the SR-IOV device plugin into a pod directly.
+
+== Supported Devices
+
+The supported NIC models in OpenShift are:
+
+- Intel 25G card with vendor id: `0x8086` and device id: `0x10fb`
+- Intel 25G card with vendor id: `0x8086` and device id: `0x1583`
+- Mellanox 25G card with vendor id: `0x15b3` and device id: `0x1015`
+- Mellanox 100G card with vendor id: `0x15b3` and device id: `0x1017`
+
+[NOTE]
+====
+For Mellanox cards, please make sure SR-IOV is enabled in firmware before provisioning VFs on host.
+====
+
+== Prerequisties
+
+[NOTE]
+====
+Please note that the creation of SR-IOV VFs are not handled by SR-IOV device plugin and SR-IOV CNI. To provision SR-IOV VF on hosts, you can either choose to configure it manually or with an upstream operator https://github.com/pliurh/node-network-operator[node-network-operator], please refer to section <<Use node network operator>>.
+====
+
+In order to use the SR-IOV network device plugin and SR-IOV CNI plugin, you'll need to run both plugins in daemon mode on each node in your cluster.
+
+Create an `openshift-sriov` namespace:
+
+[source,bash]
+----
+cat <<EOF | oc create -f -
+| apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-sriov
+  labels:
+    name: openshift-sriov
+    openshift.io/run-level: "0"
+  annotations:
+    openshift.io/node-selector: ""
+    openshift.io/description: "Openshift SR-IOV network components"
+EOF
+----
+
+Create two ServiceAccounts, `sriov-device-plugin` and `sriov-cni`:
```

Comment:
I think we need also give some permission to these service accounts?
Or else the daemonset which requeists `hostNetwork` or `hostPID` won't work.